### DISCUSSION TITLE
MobileCore.registerExtensions messes accentColor

### DISCUSSION THREAD

**Discussion by jgthoma** [adobe/aepsdk-core-ios#827] [General]
When I added Identity, Lifecycle, Signal, and UserProfile to the call to MobileCore.registerExtensions, my global accentColor no longer work. Apparently, this can be caused by setting selection color, tint color, or event creating some views before the first UIApplication window gets created by the app.

To mitigate this, I called registerExtensions with a delay.

```
    let aepExtensions = [
        Analytics.self,
        Identity.self,
        Lifecycle.self,
        Signal.self,
        UserProfile.self,
    ]

    DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
        MobileCore.registerExtensions(aepExtensions, {
            MobileCore.configureWith(appId: Self.adobeAnaltyicsKey)
        })
    }
```

**Comment by cdhoffmann** [comment]
@jgthoma Hello. What application lifecycle are you using in your app (UIApplicationDelegate or UISceneDelegate)? And where are you calling the registration code from?
